cmd/bnsd/app/app_test.go

```diff
@@ -284,9 +285,24 @@ func signAndCommit(t *testing.T, app weave_app.BaseApp, tx *app.Tx, signers []Si
 	app.EndBlock(abci.RequestEndBlock{})
 	cres := app.Commit()
 	assert.NotEmpty(t, cres.Data)
+	*height++
```

Comment:
Ahhh... you make the height a pointer, so you can increment it everytime you commit a tx.
A bit complicated in the code, but nice idea.

This could be it's own (simple) pr